src/Lecture1.hs

```diff
@@ -59,11 +61,12 @@ sumOfSquares x y = error "TODO!"
 >>> lastDigit (-17)
 7
 
-🕯 HINT: use the @mod@ function
+HINT: use the @mod@ function
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integral a => a -> a
+lastDigit n = mod (abs n) 10
```

Comment:
Absolutely correct 👍🏻 
Alternatively, you can use the infix form of a function by putting it inside backticks :slightly_smiling_face: 
```suggestion
lastDigit n = abs n `mod` 10
```